packages/js/product-editor/src/blocks/schedule-sale/edit.tsx

```diff
@@ -84,9 +87,13 @@ export function Edit( {}: BlockEditProps< ScheduleSalePricingBlockAttributes > )
 	const _dateOnSaleFrom = moment( dateOnSaleFromGmt, moment.ISO_8601, true );
 	const _dateOnSaleTo = moment( dateOnSaleToGmt, moment.ISO_8601, true );
 
-	const dateOnSaleFromGmtValidationError = useValidation(
-		'product/date_on_sale_from_gmt',
-		function dateOnSaleFromValidator() {
+	const {
+		// ref: dateOnSaleFromGmtRef,
```

Comment:
Any reason to comment this one out?

Reply:
That is because the DateTimePickerControl is not ForwardedRef yet. That is going to be covered in [#38248](https://github.com/woocommerce/woocommerce/issues/38248).